ERCS/erc-X.md

```diff
@@ -0,0 +1,550 @@
+---
+eip: X
```

Comment:
```suggestion
eip: 7826
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

Reply:
Updated!